lib/recurly/account.rb

```diff
@@ -57,6 +60,7 @@ def redemption(coupon_code)
 
     define_attribute_methods %w(
       account_code
+      account_acquisition
```

Comment:
`has_one` defines some methods on this class and this redefines them. Doing both could cause some weirdness. standard practice is to leave this out.